co-simulation/rti/mosaic-rti-core/src/main/java/org/eclipse/mosaic/rti/time/SequentialTimeManagement.java

```diff
@@ -87,6 +93,18 @@ public void runSimulation() throws InternalFederateException, IllegalValueExcept
             if (ambassador != null) {
                 federation.getMonitor().onBeginActivity(event);
                 long startTime = System.currentTimeMillis();
+
+                if (!logging_map.containsKey(event.getFederateId())) // first time printing, then directly save the last simulation time requested and print
```

Comment:
This should all be avoidable by setting log level to info. All this should be inside an if for log level debug or trace and we may want to leave a comment to consider reworking this.  What you really want is just data about when each federate progresses in time. This is probably not a long term solution.

Reply:
You could even just make this whole block a method that takes in event  and start time.

Reply:
Addressed